src/components/CompanyCard/CompanyCard.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './companyCard.scss';
+
+
+const CompanyCard = (props) => {
```

Comment:
When you're using props in your components you should add `PropTypes` to them. It'll help to create a very basic documentation of parameters as also validate arguments at runtime (only in development instance).

https://reactjs.org/docs/typechecking-with-proptypes.html

For this particular parameters you can start with 

```
const stylePropType = require('react-style-proptype');
CompanyCard .propTypes = {
  title: PropTypes.string,
  description: PropTypes.string,
  image: PropTypes.element // or elementType, I'm not sure you need to check it out
  style: stylePropType, // achievable with additional package `npm i react-style-proptype`, https://www.npmjs.com/package/react-style-proptype
};
```

Reply:
This has been corrected.